### ISSUE TITLE
can't upload manual subtitles; error: Exception on /api/episodes/subtitles [POST] and Exception on /api/movies/subtitles [POST]

### ISSUE THREAD

**Issue by xloooolx** [morpheus65535/bazarr#2072]
Describe the bug
When manually uploading subtitles to episodes or movies, I get an error,

it was fixed before 
https://github.com/morpheus65535/bazarr/issues/2051

To Reproduce

    Go to upload substitle with"srt or ass" extension
    Click on upload
    See error
    Error 500 You have disconnected from the server

Expected behavior
```
Traceback (most recent call last):
  File "/app/bazarr/bin/bazarr/../libs/flask/app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
  File "/app/bazarr/bin/bazarr/../libs/flask/app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/app/bazarr/bin/bazarr/../libs/flask_restx/api.py", line 405, in wrapper
    resp = resource(*args, **kwargs)
  File "/app/bazarr/bin/bazarr/../libs/flask/views.py", line 107, in view
    return current_app.ensure_sync(self.dispatch_request)(**kwargs)
  File "/app/bazarr/bin/bazarr/../libs/flask_restx/resource.py", line 46, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/app/bazarr/bin/bazarr/api/utils.py", line 30, in wrapper
    return actual_method(*args, **kwargs)
  File "/app/bazarr/bin/bazarr/api/episodes/episodes_subtitles.py", line 148, in post
    history_log(4, sonarrSeriesId, sonarrEpisodeId, result, fake_provider=provider, fake_score=score)
  File "/app/bazarr/bin/bazarr/sonarr/history.py", line 10, in history_log
    description = result.message
AttributeError: \'tuple\' object has no attribute \'message\
```
and this for movie 

```
Traceback (most recent call last):
  File "/app/bazarr/bin/bazarr/../libs/flask/app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
  File "/app/bazarr/bin/bazarr/../libs/flask/app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/app/bazarr/bin/bazarr/../libs/flask_restx/api.py", line 405, in wrapper
    resp = resource(*args, **kwargs)
  File "/app/bazarr/bin/bazarr/../libs/flask/views.py", line 107, in view
    return current_app.ensure_sync(self.dispatch_request)(**kwargs)
  File "/app/bazarr/bin/bazarr/../libs/flask_restx/resource.py", line 46, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/app/bazarr/bin/bazarr/api/utils.py", line 30, in wrapper
    return actual_method(*args, **kwargs)
  File "/app/bazarr/bin/bazarr/api/movies/movies_subtitles.py", line 143, in post
    history_log_movie(4, radarrId, result, fake_provider=provider, fake_score=score)
  File "/app/bazarr/bin/bazarr/radarr/history.py", line 10, in history_log_movie
    description = result.message
AttributeError: \'tuple\' object has no attribute \'message\
```

Software (please complete the following information):

Bazarr Version   1.1.5-beta.24
Package Version   v1.1.5-beta.24-ls479 by linuxserver.io
Sonarr Version   4.0.0.391
Radarr Version   4.4.3.6966
Operating System   Linux-5.15.0-58-generic-x86_64-with
Python Version   3.10.10

**Comment by morpheus65535** [comment]
Will be fixed in upcoming beta. Thank you for reporting! :-)

**Comment by namniav** [comment]
> Will be fixed in upcoming beta. Thank you for reporting! :-)

Hi, with v1.3.0 which includes [2acf245](https://github.com/morpheus65535/bazarr/commit/2acf2451b44e0dd407e299170ed37117b0eeff7b), I got 
* `Error 500:You have disconnected from the server` from web-ui, and
*  `Exception on /api/episodes/subtitles [POST]` from logs.

With debug logging, I saw
```
File "bazarr/api/episodes/episodes_subtitles.py", line 144, in post
    raise ValueError('A subtitle of an invalid format was uploaded.')
```
and realized that those subtitle files have no filename extensions. They were successfully uploaded after renamed with correct extensions.

Leaving this message here in case you guys would like to improve the web-ui tips.
